mongomock/aggregate.py

```diff
@@ -543,6 +544,41 @@ def _handle_type_convertion_operator(self, operator, values):
                 raise TypeError("'%s' type is not supported" % type(parsed))
             return decimal_value
 
+        # Document: https://docs.mongodb.com/manual/reference/operator/aggregation/arrayToObject/
+        if operator == '$arrayToObject':
+            try:
+                parsed = self.parse(values)
+            except KeyError:
+                return None
+
+            if parsed is None:
+                return None
+
+            if not isinstance(parsed, list):
```

Comment:
In general were `list` is supported, `tuple` is supported as well. Can you check on `pymongo` and update it here please?